tests/test_ensemble.py

```diff
@@ -27,8 +27,10 @@ def test_single_model(self):
 
         m = OneInputOneOutputOneEventLM()
         
-        with self.assertRaises(ValueError):
+        try:
             EnsembleModel([m])
+        except ValueError:
+            self.fail("Error EnsembleModel raised ValueError Unexpectedly")
 
```

Comment:
Since this is now an acceptable state, we want to add tests to make sure it behaves as expected. We should add something similar to what test_two_models_identical or test_two_models_different tests do for two models. 

Let me know if you would like any help with that